internal/services/fabric/fabric_capacity_resource.go

```diff
@@ -0,0 +1,318 @@
+	return &FabricCapacityResource{}
+}
+
+func (r FabricCapacityResource) IDValidationFunc() pluginsdk.SchemaValidateFunc {
+	return fabriccapacities.ValidateCapacityID
+}
+
+func (r FabricCapacityResource) ResourceType() string {
+	return "azurerm_fabric_capacity"
+}
+
+func (r FabricCapacityResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringMatch(
+				regexp.MustCompile(`^[a-z]([a-z\d]{1,61}[a-z\d])$`),
+				"`name` must be between 3 and 63 characters. It can contain only lowercase letters and numbers. It must start with a lowercase letter.",
+			),
+		},
+
+		"location": commonschema.Location(),
+
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		"sku": {
+			Type:     pluginsdk.TypeList,
+			Required: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"name": {
+						Type:         pluginsdk.TypeString,
+						Required:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
+					},
+
+					"tier": {
+						Type:     pluginsdk.TypeString,
+						Required: true,
+						ValidateFunc: validation.StringInSlice([]string{
+							string(fabriccapacities.RpSkuTierFabric),
+						}, false),
+					},
+				},
+			},
+		},
+
+		"administration_members": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Schema{
+				Type:         pluginsdk.TypeString,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+		},
+
+		"tags": commonschema.Tags(),
+	}
+}
+
+func (r FabricCapacityResource) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{}
+}
+
+func (r FabricCapacityResource) Create() sdk.ResourceFunc {
+	return sdk.ResourceFunc{
+		Timeout: 30 * time.Minute,
+		Func: func(ctx context.Context, metadata sdk.ResourceMetaData) error {
+			client := metadata.Client.Fabric.FabricCapacitiesClient
+			subscriptionId := metadata.Client.Account.SubscriptionId
+
+			var model FabricCapacityResourceModel
+			if err := metadata.Decode(&model); err != nil {
+				return fmt.Errorf("decoding: %+v", err)
+			}
+
+			id := fabriccapacities.NewCapacityID(subscriptionId, model.ResourceGroupName, model.Name)
+			existing, err := client.Get(ctx, id)
+			if err != nil && !response.WasNotFound(existing.HttpResponse) {
+				return fmt.Errorf("checking for existing %s: %+v", id, err)
+			}
+
+			if !response.WasNotFound(existing.HttpResponse) {
+				return metadata.ResourceRequiresImport(r.ResourceType(), id)
+			}
+
+			properties := fabriccapacities.FabricCapacity{
+				Location:   location.Normalize(model.Location),
+				Properties: fabriccapacities.FabricCapacityProperties{},
+			}
+
+			if len(model.AdministrationMembers) == 0 {
+				return fmt.Errorf("`administration_members` is required when creating a fabric capacity")
```

Comment:
`administration_members` should be `required`. At least one admin member is expected.

Reply:
Set `administration_members ` to Optional and add a required check in create function. This is because after verifying the behavior of the API, it was found that `administration_members` is required when creating resource, but `administration_members` can be unspecified or cleared when updating resource. In addition, `administration_members` can also be deleted on the Azure Portal.